docs/getting-started/quick-start-watt.md

```diff
@@ -292,7 +292,7 @@ and the `path` in `web/composer/platformatic.json` accordingly.
 
 ## `fetch` the data from the Node.js app in the Next.js app
 
-Replace `web/next/src/app/page.js`, with the following code:
+Replace `web/next/app/page.js`, with the following code:
```

Comment:
This change is correct dependening on the choices picked when creating a Next app.

Reply:
Ah, I just went with the default selections and it was wrong for that.